lib/galaxy/datatypes/binary.py

```diff
@@ -236,6 +236,49 @@ def sniff(self, filename):
         return False
 
 
+class Bref3(Binary):
+    """Bref3 format is a binary format for storing phased, non-missing genotypes for a list of samples."""
```

Comment:
if you have a URL where this format is described please include it

Reply:
Yes